### ISSUE TITLE
azure-ai-ml content validation issue for learn microsoft website.

### ISSUE THREAD

**Issue by zedy-wj** [zedy-wj/content-validation#192]
1.
**ErrorInfo**: Some cells in the table are missing content
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl?view=azure-python
**ErrorLocation**:
 - Appears 3 times , location : https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl?view=azure-python#enums

**Note**: Some cells in the table are missing content - this type of issues appears 219 times, currently only one is shown here. For more details, please click on the excel download link below to view.

2.
**ErrorInfo**: The test link has garbled text.
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl?view=azure-python
**ErrorLocation**:
 - Appears 1 times , garbled text :   :keyword early_termination:
 - Appears 1 times , garbled text :   :paramtype early_termination:

**Note**: The test link has garbled text. - this type of issues appears 32 times, currently only one is shown here. For more details, please click on the excel download link below to view.

3.
**ErrorInfo**: Extra label found: <xref
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml?view=azure-python
**ErrorLocation**:
 - Appears 24 times , label : <xref 

**Note**: Extra label found: <xref - this type of issues appears 141 times, currently only one is shown here. For more details, please click on the excel download link below to view.

4.
**ErrorInfo**: Unnecessary symbols found: "[" ,"~"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "[" in text: ] :keyword early_termination: Type of early termination policy. :paramtype early_termination: Union[
 - Appears 3 times ,  Unnecessary symbol: "~" in text: ~azure.mgmt.machinelearningservices.models.BanditPolicy, ~azure.mgmt.machinelearningservices.models.MedianStoppingPolicy, ~azure.mgmt.machinelearningservices.models.TruncationSelectionPolicy

5.
**ErrorInfo**: Unnecessary symbols found: "[" ,"~"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl.imagesweepsettings?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "[" in text: ] :keyword early_termination: Type of early termination policy. :paramtype early_termination: Union[
 - Appears 3 times ,  Unnecessary symbol: "~" in text: ~azure.mgmt.machinelearningservices.models.BanditPolicy, ~azure.mgmt.machinelearningservices.models.MedianStoppingPolicy, ~azure.mgmt.machinelearningservices.models.TruncationSelectionPolicy
 - Appears 1 times ,  Unnecessary symbol: "[" in text: Union[ str, GRID, BAYESIAN, RANDOM

6.
**ErrorInfo**: Unnecessary symbols found: "<"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl.forecastingjob?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "<" in text: When forecasting, this parameter represents n historical periods to use to generate forecasted values, <= training set size. If omitted, n is the full training set size. Specify this parameter when you only want to consider a certain amount of history when training the model. If set to 'auto', rolling window will be estimated as the last value where the PACF is more then the significance threshold. Please see target_lags section for details.

7.
**ErrorInfo**: Unnecessary symbols found: "<"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl.forecastingsettings?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "<" in text: When forecasting, this parameter represents n historical periods to use to generate forecasted values, <= training set size. If omitted, n is the full training set size. Specify this parameter when you only want to consider a certain amount of history when training the model. If set to 'auto', rolling window will be estimated as the last value where the PACF is more then the significance threshold. Please see target_lags section for details.

8.
**ErrorInfo**: Unnecessary symbols found: "~"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl.imageclassificationjob?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "~" in text: List[~azure.ai.ml.automl.ImageClassificationSearchSpace] :return: Search space for AutoML Image Classification and Image Classification Multilabel tasks.
 - Appears 1 times ,  Unnecessary symbol: "~" in text: ~azure.ai.ml.automl.ImageModelSettingsClassification :return: Training parameters for AutoML Image Classification and Image Classification Multilabel tasks.

**Note**: Unnecessary symbols found: "~" - this type of issues appears 20 times, currently only one is shown here. For more details, please click on the excel download link below to view.

9.
**ErrorInfo**: Unnecessary symbols found: "["
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl.imageinstancesegmentationjob?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "[" in text: Overlap ratio between adjacent tiles in each dimension. Must be float in the range [0, 1).

**Note**: Unnecessary symbols found: "[" - this type of issues appears 5 times, currently only one is shown here. For more details, please click on the excel download link below to view.

10.
**ErrorInfo**: Unnecessary symbols found: "~" ,"["
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.automl.imageobjectdetectionsearchspace?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "~" in text: Initial learning rate. Must be a float in the range [0, 1]. :type learning_rate: float or ~azure.ai.ml.entities.SweepDistribution
 - Appears 1 times ,  Unnecessary symbol: "~" in text: Maximum size of the image to be rescaled before feeding it to the backbone. Must be a positive integer. Note: training run may get into CUDA OOM if the size is too big. Note: This settings is not supported for the 'yolov5' algorithm. :type max_size: int or ~azure.ai.ml.entities.SweepDistribution
 - Appears 1 times ,  Unnecessary symbol: "[" in text: Overlap ratio between adjacent tiles in each dimension. Must be float in the range [0, 1).

11.
**ErrorInfo**: Unnecessary symbols found: "~" ,"]" ,"&"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities?view=azure-python
**ErrorLocation**:
 - Appears 2 times ,  Unnecessary symbol: "~" in text: vartype system_data: ~azure.ai.ml.entities.SystemData
 - Appears 2 times ,  Unnecessary symbol: "~" in text: Feature window :keyword feature_window_end: Specifies the feature window end time. :paramtype feature_window_end: ~datetime.datetime :keyword feature_window_start: Specifies the feature window start time. :paramtype feature_window_start: ~datetime.datetime
 - Appears 1 times ,  Unnecessary symbol: "~" in text: This class should not be instantiated directly. Instead, use the child class ~azure.ai.ml.entities.CommandComponent.
 - Appears 3 times ,  Unnecessary symbol: "]" in text: ]]
 - Appears 1 times ,  Unnecessary symbol: "&" in text: This class is used to organize and parse diagnostics from both client & server side before expose them. The result is immutable.
 - Appears 1 times ,  Unnecessary symbol: "~" in text: Note: For connections to OpenAI, Cognitive Search, and Cognitive Services, use the respective subclasses (ex: ~azure.ai.ml.entities.OpenAIConnection) instead of instantiating this class directly.

12.
**ErrorInfo**: Unnecessary symbols found: "]"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.command?view=azure-python
**ErrorLocation**:
 - Appears 4 times ,  Unnecessary symbol: "]" in text: ]]]

13.
**ErrorInfo**: Unnecessary symbols found: "]"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.spark?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "]" in text: ]]

14.
**ErrorInfo**: Unnecessary symbols found: "]"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.sweep?view=azure-python
**ErrorLocation**:
 - Appears 2 times ,  Unnecessary symbol: "]" in text: ]]

15.
**ErrorInfo**: Unnecessary symbols found: "<" ,">"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.packageinputpathid?view=azure-python
**ErrorLocation**:
 - Appears 5 times ,  Unnecessary symbol: "<" in text: The resource ID of the input path. e.g. "azureml://subscriptions/<>/resourceGroups/ <>/providers/Microsoft.MachineLearningServices/workspaces/<>/data/<>/versions/<>".
 - Appears 5 times ,  Unnecessary symbol: ">" in text: The resource ID of the input path. e.g. "azureml://subscriptions/<>/resourceGroups/ <>/providers/Microsoft.MachineLearningServices/workspaces/<>/data/<>/versions/<>".

16.
**ErrorInfo**: Unnecessary symbols found: "<" ,">"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.packageinputpathurl?view=azure-python
**ErrorLocation**:
 - Appears 4 times ,  Unnecessary symbol: "<" in text: The url of the input path. e.g. "azureml://subscriptions/<>/resourceGroups/ <>/providers/Microsoft.MachineLearningServices/workspaces/data/<>/versions/<>".
 - Appears 4 times ,  Unnecessary symbol: ">" in text: The url of the input path. e.g. "azureml://subscriptions/<>/resourceGroups/ <>/providers/Microsoft.MachineLearningServices/workspaces/data/<>/versions/<>".

17.
**ErrorInfo**: Unnecessary symbols found: "&"
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.entities.validationresult?view=azure-python
**ErrorLocation**:
 - Appears 1 times ,  Unnecessary symbol: "&" in text: This class is used to organize and parse diagnostics from both client & server side before expose them. The result is immutable.

18.
**ErrorInfo**: Missing Type Annotation
**ErrorLink**: https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.data_transfer.datatransfercopy?view=azure-python
**ErrorLocation**:
 - Method name: get,    arguments:  key ;    
 - Method name: pop,    arguments:  k ;    [d] ;    
 - Method name: setdefault,    arguments:  key ;    
 - Method name: update,    arguments:  [E] ;    **F ;    

**Note**: Missing Type Annotation - this type of issues appears 12 times, currently only one is shown here. For more details, please click on the excel download link below to view.
